src/libs/ajax/data-table-providers/DataTableProvider.ts

```diff
@@ -50,6 +50,36 @@ export interface EntityQueryOptions {
   filterOperator: string
 }
 
+export type UploadParameters = {
+  file: File
+  useFireCloudDataModel: boolean
+  deleteEmptyValues: boolean
+  namespace: string
+  name: string
+  workspaceId: string
+  recordType: string
+}
+
+export type isInvalidTsvOptions = {
+  modeMatches: boolean
+  filePresent: boolean
+  match: boolean
```

Comment:
Could this be more descriptive? What's matching what?

Reply:
I should've been more clear: By "this" in my previous comment, I'd meant the property `match`. By looking at the definition of `InvalidTsvOptions`, I can't tell what may `match` what.